docs/user_guide/light-dark.rst

```diff
@@ -0,0 +1,129 @@
+
+.. _manage-themes:
+
+Light and dark themes
+=====================
+
+You can change the major background / foreground colors of this theme according to "dark" and "light" modes.
+These are controlled by a button in the navigation header, with the following options:
+
+- A ``light`` theme with a bright background and dark text / UI elements
+- A ``dark`` theme with a dark background and light text / UI elements
+- ``auto``: the documentation theme will follow the system default that you have set
+
+
+Configure default theme mode
+----------------------------
+
+By default, visitors to your documentation will use the theme mode ``auto``.
+This will choose a theme based on the user's system settings, and default to ``light`` if not settings are present.
+
+If you wish to use a different default theme mode, set the ``default_mode`` configuration to one of ``auto``, ``dark``, ``light``.
+For example:
+
+.. code-block:: python
+
+   html_context = {
+      # ...
+      "default_mode": "light"
+   }
+
+For more information, see :ref:`manage-themes`.
+
+.. tip::
+
+   To completely remove the theme management, configure ``default_mode`` to the value you want in your documentation (``light`` or ``dark``) and then remove the theme-switcher from the ``navbar_end`` section of the header navbar configuration:
+
+   .. code-block:: python
+
+      html_theme_options {
+          # ...
```

Comment:
```suggestion
          ...
```
for consistency (elsewhere we don't comment out the `...`, even though it's technically not valid Python)